src/main.rs

```diff
@@ -59,7 +61,7 @@ async fn main() -> Result<(), Box<dyn Error>> {
     let host = IpAddr::from_str(&host_raw)?;
     let port = env::var("PORT").unwrap_or_else(|_| "80".into()).parse()?;
 
-    let client = Client::new(env::var("DISCORD_TOKEN")?);
+    let client_map = Arc::new(ClientMap::new(env::var("DISCORD_TOKEN")?));
```

Comment:
Out of scope, but having the `DISCORD_TOKEN` env be optional would be nice

Reply:
Won't do that since this PR tries to keep performance for users with only one token equal to previously. As you mentioned, out of scope and therefore I won't change it